applications/machine_learning/README.rst

```diff
@@ -79,6 +79,30 @@ By default, the application uses pretrained machine leaning models deployed in `
 The application displays LED effects that correspond to the machine learning results.
 For more detailed information, see the `User interface`_ section.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the nRF Machine Learning application, you can suspend or turn off modules if they are not in use.
+The application uses :ref:`caf_power_manager` module for this purpose, which means that Zephyr power management is forced to :c:enumerator:`PM_STATE_ACTIVE` when device is in either the active or the suspended state, but the power off state is forced directly by :ref:`caf_power_manager` as Zephyr's :c:enumerator:`PM_STATE_SOFT_OFF`.
+
+* In the :c:enumerator:`PM_STATE_ACTIVE` state the device is in working condition, Bluetooth is advertising if required, all the connections are maintained.
+* In the :c:enumerator:`POWER_MANAGER_LEVEL_SUSPENDED` state, the device maintains the active Bluetooth connection.
+* In the :c:enumerator:`POWER_MANAGER_LEVEL_OFF` state, the CPU is switched to the off mode.
+
+In both cases, most of the functionalities are disabled.
+For example, LEDs and sensors are turned off and Bluetooth advertising is stopped.
+
+Any button press can wake up the device.
+
+For the Thingy:53, the sensor supports a trigger that can be used for active power management.
+As long as the device detects acceleration, the board is kept in the active state.
+When the board is in :c:enumerator:`POWER_MANAGER_LEVEL_SUSPENDED` state, it can be woken up by acceleration threshold, that is by moving the device.
```

Comment:
```suggestion
When the board is in the :c:enumerator:`POWER_MANAGER_LEVEL_SUSPENDED` state, it can be woken up by acceleration threshold by moving the device.
```